src/core/RandomVariables.jl

```diff
@@ -54,30 +54,25 @@ export  VarName,
 """
 ```
 struct VarName{sym}
-    csym      ::    Symbol
     indexing  ::    String
-    counter   ::    Int
 end
 ```
 
-A variable identifier. Every variable has a symbol `sym`, indices `indexing`, and
-internal fields: `csym` and `counter`. The Julia variable in the model corresponding to
-`sym` can refer to a single value or to a hierarchical array structure of univariate,
-multivariate or matrix variables. `indexing` stores the indices that can access the
-random variable from the Julia variable.
+A variable identifier. Every variable has a symbol `sym` and `indices `indexing`. 
+The Julia variable in the model corresponding to `sym` can refer to a single value or 
+to a hierarchical array structure of univariate, multivariate or matrix variables. `indexing` stores the indices that can access the random variable from the Julia 
+variable.
 
 Examples:
 
 - `x[1] ~ Normal()` will generate a `VarName` with `sym == :x` and `indexing == "[1]"`.
 - `x[:,1] ~ MvNormal(zeros(2))` will generate a `VarName` with `sym == :x` and
- `indexing == "[Colon(), 1]"`.
+ `indexing == "[Colon(),1]"`.
 - `x[:,1][2] ~ Normal()` will generate a `VarName` with `sym == :x` and
- `indexing == "[Colon(), 1][2]"`.
+ `indexing == "[Colon(),1][2]"`.
 """
 struct VarName{sym}
-    csym      ::    Symbol        # symbol generated in compilation time
-    indexing  ::    String        # indexing
-    counter   ::    Int           # counter of same {csym, uid}
```

Comment:
Seems that here we remove two fileds. Are they in favour of https://github.com/TuringLang/Turing.jl/issues/972#issuecomment-555117648?

Reply:
These 2 are independent. The 2 fields removed were simply never used in all of Turing. The custom variable name was implemented using `NamedDist`.